### PR TITLE
 RE-1163 Add rpc-openstack master tox jobs

### PR DESCRIPTION
**Pull request by git-harry**
Issue: [RE-1163](https://rpc-openstack.atlassian.net/browse/RE-1163)

### CODE REVIEW

**Comment by git-harry** [comment]
recheck_xenial_functional_test

File: rpc_jobs/rpc_openstack.yml [hunk 1]
```diff
@@ -18,6 +18,27 @@
...
+      - "master"
+    image:
+      - container:
+          SLAVE_TYPE: "container"
+    scenario:
+      - "ansible-lint"
+      - "ansible-syntax"
+      - "bashate"
+      - "pep8"
+      - "releasenotes"
```
Comment (hughsaunders) [inline rpc_jobs/rpc_openstack.yml]: What does this do, and do we need to have this as well as the [release PR job](https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/standard_job_premerge_release.yml)?

Comment (git-harry) [reply rpc_jobs/rpc_openstack.yml]: https://github.com/rcbops/rpc-openstack/blob/master/tox.ini#L58 - this looks to be doing something different.

This change is simply moving the existing tests. I'd be inclined to leave any modification the tests to a separate task.

Comment (hughsaunders) [reply rpc_jobs/rpc_openstack.yml]: I'm so glad we have two totally separate ways of generating release notes. That can be addressed in a different issue.

**Pull request merged**